src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="/styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img src="/images/imac.jpeg"
+           alt="imac"
+           class="card__images"
+      >
```

Comment:
```suggestion
      <img 
        src="/images/imac.jpeg"
        alt="imac"
        class="card__images"
      >
```
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/37ed1c5b-78da-4e04-a9c8-15a361c4a92b)
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/6c7d90d9-1927-4679-829c-dfc3b2bc5ef1)

Reply:
changed